IdokladSdk.IntegrationTests/Tests/Clients/ReceivedInvoice/ReceivedInvoiceTests.Async.cs

```diff
@@ -107,6 +107,42 @@ public async Task CopyAsync_SuccessfullyGetPosModel()
 
         [Test]
         [Order(13)]
+        public async Task UpdateAsync_AddNewItems_Sucessfull()
+        {
+            // Arrange
+            var invoiceToUpdate = (await _receivedInvoiceClient.Detail(_receivedInvoiceIdAsync).GetAsync()).AssertResult();
+            var itemName2 = "Test2Test";
+            var itemName3 = "Test3Test";
+            var model = new ReceivedInvoicePatchModel
+            {
+                Id = invoiceToUpdate.Id,
+                Items = new List<ReceivedInvoiceItemPatchModel>
+                {
+                    new ReceivedInvoiceItemPatchModel
+                    {
+                        Id = 1,
```

Comment:
Proč tomu přiřazuješ Id = 1? A proč druhé položce žádné?

Reply:
je to z časti aj test na API, lebo keď mApp nám poslala ich dočasné Id (napr. hodnoty 1, 2, 3), tak to u nás spadlo na Internal Server Error (toto je už na API opravené) a chcel som to podchytiť aj v tomto teste. Zároveň som chcel týmto testom ukázať, že Id sa pre nový item v patchi nemusí vyplniť